### PR TITLE
Issue 1376

### PR DESCRIPTION
**Pull request by PiotrKozlowski**
for: https://github.com/openstax/unified/issues/1376

### CODE REVIEW

File: src/app/components/ScrollLock.tsx [hunk 1]
```diff
@@ -36,8 +37,16 @@ const fadeIn = keyframes`
...
+interface OverlayProps {
+  className?: string;
+  [key: string]: any;
+}
+
 // tslint:disable-next-line:variable-name
-export const Overlay = styled.div`
+export const Overlay = styled(({ className, ...otherProps }: OverlayProps) => {
+  useDisableContentTabbing();
+  return <div className={className} {...otherProps} />;
```
Comment (TomWoodward) [inline src/app/components/ScrollLock.tsx:48]: you don't have to split out the classname it could be applied the same way as `otherProps`

File: src/app/content/highlights/components/HighlightsPopUp.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { HTMLElement } from '@openstax/types/lib.dom';
 import React from 'react';
+import { createPortal } from 'react-dom';
```
Comment (TomWoodward) [inline src/app/content/highlights/components/HighlightsPopUp.tsx:3]: i was hoping we would do the portal stuff in some kind of shared modal component and not have to do it in 4 places

File: src/app/content/practiceQuestions/components/PracticeQuestionsPopup.tsx [hunk 3]
```diff
@@ -73,7 +74,8 @@ const PracticeQuestionsPopup = () => {
         </Header>
         <ShowPracticeQuestions />
       </Modal>
-    </PopupWrapper>
+    </PopupWrapper>,
+    assertDocument().body
```
Comment (TomWoodward) [inline src/app/content/practiceQuestions/components/PracticeQuestionsPopup.tsx:78]: also i'm not sure you're supposed to portal directly into the body, i think you might be supposed to create a mount component and add it to the body and portal into that, but i'm not totally sure, if you saw this in the docs then its fine

File: src/app/reactUtils.ts [hunk 4]
```diff
@@ -191,3 +191,55 @@ export const useOnScrollTopOffset = () => {
 
   return topOffset;
 };
+
+const tabbableElementsSelector = [
```
Comment (TomWoodward) [inline src/app/reactUtils.ts:195]: if you got this list from somewhere please link it in a comment

**Review by TomWoodward — CHANGES_REQUESTED** [review]

File: src/app/content/practiceQuestions/components/PracticeQuestionsPopup.tsx (hunk 3 above)
Comment (PiotrKozlowski) [reply src/app/content/practiceQuestions/components/PracticeQuestionsPopup.tsx:78]: I've neverd heard of this and docs just says:
```
render() {
  // React does *not* create a new div. It renders the children into `domNode`.
  // `domNode` is any valid DOM node, regardless of its location in the DOM.
  return ReactDOM.createPortal(
    this.props.children,
    domNode  );
}
```

There is also closed issue on Material-UI repo according to this
https://github.com/mui-org/material-ui/issues/21626

Comment (TomWoodward) [reply src/app/content/practiceQuestions/components/PracticeQuestionsPopup.tsx]: /shrug good enough for material-ui good enough for me

File: src/app/content/highlights/components/SummaryPopup/HighlightsHelpInfo.tsx [hunk 5]
```diff
@@ -32,6 +32,7 @@ const slideInAnimation = css`
 const Wrapper = styled.div`
   position: fixed;
   bottom: 0;
+  left: 0;
```
Comment (TomWoodward) [inline src/app/content/highlights/components/SummaryPopup/HighlightsHelpInfo.tsx:35]: why is this added here?

**Review by TomWoodward — APPROVED** [review]
i'm not sure about that one style change, but everything else looks good

File: src/app/content/highlights/components/SummaryPopup/HighlightsHelpInfo.tsx (hunk 5 above)
Comment (PiotrKozlowski) [reply src/app/content/highlights/components/SummaryPopup/HighlightsHelpInfo.tsx:35]: `HighlightsHelpInfo` was previously outside of the `<Modal>` and now it is inside
